tests/pyincore/analyses/housingrecoveryserial/test_housingrecoverysequential_hhinc.py

```diff
@@ -0,0 +1,73 @@
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+import os
+
+from pyincore import IncoreClient, HHRSOutputProcess, Dataset
+from pyincore.analyses.housingrecoverysequential import HousingRecoverySequential
+from timeit import default_timer as timer
+import pyincore.globals as pyglobals
+
+
+def run_with_base_class():
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+
+    start = timer()
+
+    # dev Joplin testbed
+    population_dislocation = Dataset.from_file(
```

Comment:
Can we make this a dataset on dev? This is a pretty big file and is better served from the incore-dev and referenced by ID instead.